base/linalg/lapack.jl

```diff
@@ -664,7 +675,7 @@ for (gesvx, elty) in
             ferr      = similar(A, $elty, nrhs)
             berr      = similar(A, $elty, nrhs)
             work      = Array($elty, 4n)
-            iwork     = Array($elty, n)
+            iwork     = Array($elty, 2n)
```

Comment:
Which test is causing the segfault. So far I haven't been able to make it segfault with `length(iwork)=n`.